client/src/components/protected/ChatBox.js

```diff
@@ -1,73 +1,99 @@
+    });
+  };
 
   increment = (e) => {
     if (this.state.num >= 9) return;
     this.setState({ num: parseInt(this.state.num) + 1 });
-  }
+  };
 
   decrement = (e) => {
     if (this.state.num === 0) return;
     this.setState({ num: parseInt(this.state.num) - 1 });
-  }
+  };
 
   sendCurrentMsg = () => {
-    const { num, word, messages } = this.state
-    if (num === '' || word === '') {
-      return
+    const { num, word, messages } = this.state;
+    if (num === "" || word === "") {
+      return;
     }
-    this.props.submitHint({ num, word })
-    const newMessage = `${num} - ${word}`
+    this.props.submitHint({ num, word });
+    const newMessage = `${num} - ${word}`;
 
-    this.setState({ num: '1', word: '', messages: [...messages, newMessage] })
-  }
+    this.setState({ num: "1", word: "", messages: [...messages, newMessage] });
+  };
 
   render() {
-    const { num, word, messages } = this.state
+    const { num, word, messages } = this.state;
     const text = messages.map((step, index) => {
-      return (<ListItem key={`msg-${index}`}>{step}</ListItem>);
+      return <ListItem key={`msg-${index}`}>{step}</ListItem>;
     });
     const { classes } = this.props;
     return (
       <Paper className={classes.chatBox}>
-        <List className={classes.chatList}>
-          {text}
-        </List>
-        <Grid container item className={classes.inputStyle}>
-          <Input className={classes.inputBox} name="word" value={word} onChange={this.handleChange} />
+        <List className={classes.chatList}>{text}</List>
+        <Grid container className={classes.inputStyle}>
+          <Input
+            className={classes.inputBox}
+            name="word"
+            value={word}
+            onChange={this.handleChange}
+          />
           <Typography variant="h5">
-            <Button disabled={this.state.num <= 1 ? true : false} className={"MuiPaper-elevation1 "} onClick={this.decrement}>-</Button>
+            <Button
+              disabled={this.state.num <= 1 ? true : false}
+              className={"MuiPaper-elevation1 "}
+              onClick={this.decrement}
+            >
+              -
+            </Button>
             {num}
-            <Button disabled={this.state.num >= 9 ? true : false} className={"MuiPaper-elevation1"} onClick={this.increment}>+</Button>
+            <Button
+              disabled={this.state.num >= 9 ? true : false}
+              className={"MuiPaper-elevation1"}
+              onClick={this.increment}
+            >
+              +
+            </Button>
           </Typography>
-        </Grid>
-        <Button onClick={() => this.sendCurrentMsg()} color='primary' variant='contained'>
-          Submit Hint
+          <Button
+            size="small"
+            onClick={() => this.sendCurrentMsg()}
+            color="primary"
+            variant="contained"
+          >
+            Submit Hint
           </Button>
+        </Grid>
```

Comment:
I didn't change things here, but you will want to fix this, in small screens the button goes past the container and mess up the layout